api/qsharp/microsoft.quantum.diagnostics.assertqubitisinstatewithintolerance.yml

```diff
@@ -26,7 +26,7 @@ remarks: |-
 
   The tolerance is
   the $L\_{\infty}$ distance between 3 dimensional real vector (x₂,x₃,x₄) defined by
-  $\langle\psi|\psi\rangle = x\_1 I + x\_2 X + x\_3 Y + x\_4 Z$ and real vector (y₂,y₃,y₄) defined by
```

Comment:
These files are automatically overwritten from API documentation comments in the source code itself. These changes are useful, such that I want to make sure we capture them; would you be willing to apply your changes for the API reference to the Q# sources in Microsoft/QuantumLibraries instead? Thanks!

Reply:
Absolutely! I'll let you know if I have any trouble branching off that (in case I'm not listed as a contributor to that repo as well). While I'm at it, would you like me to change the expressions currently in unicode(?) style to latex as well? 
e.g. in this specific paragraph alone:
![Capture](https://user-images.githubusercontent.com/47889725/64279242-e05b7680-cf4e-11e9-94ea-65579e6de253.JPG)
I'm not sure how widespread it is throughout the docs, but it's a smidge unsightly/cumbersome.

Reply:
Thanks! As for Unicode vs TeX notation, our general strategy has been to use Unicode only where LaTeX notation isn't supported (e.g.: in samples and VS / VSCode–based katas). The `/// # Summary` section of API documentation comments is used in the widest variety of contexts, such that math notation should be avoided there as best as possible (`/// # Description` and `/// # Remarks` are preferred sections for mathematical content).

Reply:
That makes sense, thanks for the clarification. New point of confusion though: I can't find anything in Microsoft/QuantumLibraries that needs updating. Even searching for "\langle," "\rangle," and "\mathbb" merely returned a few spots that I hadn't updated in the first place because they're not technically bras or kets, they just happen to use the angle brackets (e.g. "<ZZI, IZZ>" referring to group generators). 

Not sure if I'm missing something, but I checked out and searched the whole repo in Visual Studio. Just let me know!

Reply:
Ah, my apologies, I had forgotten that AssertQubitIsInStateWithinTolerance is one of the several functions and operations that gets pulled in from the runtime and not the standard libraries. This YAML file is generated from the docs at https://github.com/microsoft/qsharp-runtime/blob/ebb4b2a9de0110ab45a6e38b38cfbd664759483f/src/Simulation/Intrinsic/Diagnostics/AssertQubit.qs#L104. Thanks!